integration-tests/utils/integration-utils.js

```diff
@@ -213,25 +213,6 @@ const dismissNotifications = async (page) => {
   return !!notificationCloseButtons.length && delay(1000); // delayed for alerts to animate off
 };
 
-const dismissNPSSurvey = async (page) => {
```

Comment:
I think this might be to dismiss the AppCues NPS survey.

Reply:
Wait I thought appcues and NPS were different…

Reply:
There's still an NPS survey ("How likely are you to recommend Terra..."), but it's done via AppCues now.

This function clicks on an "Ask Me Later" button in an iframe labelled "NPS Survey", which doesn't match the old component in NpsSurvey.js.

Reply:
ah I see! Thanks for catching that, I will put this back and update comments to make it clearer that it's going through appcues